web/src/index.ejs

```diff
@@ -1,139 +1,146 @@
-        href="/fonts/open-sans/open-sans-v15-greek_cyrillic-ext_vietnamese_latin_cyrillic_greek-ext-700.woff2" as="font"
-        type="font/woff2" crossorigin />
-  <link rel="preload"
-        href="/fonts/open-sans/open-sans-v15-greek_cyrillic-ext_vietnamese_latin_cyrillic_greek-ext-regular.woff2"
-        as="font" type="font/woff2" crossorigin />
-
-  <link rel="stylesheet" href="/fonts/raleway/raleway.css" type="text/css" />
-  <link rel="preload" href="/fonts/raleway/raleway-v12-latin_latin-ext-700.woff2" as="font" type="font/woff2"
-        crossorigin />
-  <link rel="preload" href="/fonts/raleway/raleway-v12-latin_latin-ext-regular.woff2" as="font" type="font/woff2"
-        crossorigin />
-
-  <link rel="icon" type="image/png"  sizes="16x16" href="<%= config.icons.favicons %>favicon-16x16.png">
-  <link rel="icon" type="image/png"  sizes="48x48" href="<%= config.icons.favicons %>favicon-48x48.png">
-  <link rel="icon" type="image/png"  sizes="96x96" href="<%= config.icons.favicons %>favicon-96x96.png">
-  <link rel="icon" type="image/png"  sizes="192x192" href="<%= config.icons.favicons %>favicon-192x192.png">
-  <link rel="icon" type="image/png"  sizes="512x512" href="<%= config.icons.favicons %>favicon-512x512.png">
-
-  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="<%= config.icons.appleTouchIcon %>">
-  <link rel="shortcut icon" href="<%= config.icons.favicons %>favicon.ico">
-
-  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="<%= config.lightTheme.colors.themeColor %>">
-
-  <style><% if (config.splashScreen) { %>
-    @keyframes show {
-      0% {
-        opacity: 0;
-      }
-      86% {
-        opacity: 0;
-      }
-      100% {
-        opacity: 1;
-      }
-    }
-
-    #splash {
-      position: fixed;
-      top: 0;
-      left: 0;
-      z-index: 1;
-      height: 100vh;
-      width: 100vw;
-
-      animation: show 1.5s forwards;
-      pointer-events: none;
-      transition: height 0s 0.2s;
-    }
-
-    #splash > div {
-      background-color: <%= config.splashScreen.backgroundColor %>;
-      width: 100%;
-      height: 100%;
-      display: flex;
-      justify-content: center;
-      align-items: center;
-      transition: opacity 0.2s ease-in;
-    }
-
-    #splash.splash-hidden {
-      animation-play-state: paused;
-      height: 0;
-    }
-
-    .splash-hidden > div {
-      opacity: 0;
-    }
-
-    #splash img {
-      height: 50%;
-      max-width: 100%;
-      object-fit: contain;
-    }
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title><%= config.appName %></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <% if (config.manifestUrl) { %>
+        <link rel="manifest" href="<%= config.manifestUrl %>">
     <% } %>
-    .no-js {
-      position: fixed;
-      z-index: 2;
-      text-align: center;
-      padding: 20px;
-      top: 0;
-      left: 0;
-      right: 0;
-    }
-  </style>
+
```

Comment:
What did happen here?

Reply:
i reformatted the file, seems liek noone did that before. gonna mark the changes